inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -18,16 +18,16 @@
 #}
 
 {% macro render_record_authors(number_of_displayed_authors) %}
-  {% if record.get('authors')|length > 0 %}
+  {% if record.get('authors', [])|length > 0 %}
```

Comment:
we can move to the more simple syntax:

```python
{% if record.authors | length > 0 %}
```

if the key does not exist jinja2 does not crash and `length` returns 0.